docs/extensibility/extending-blocks.md

```diff
@@ -333,9 +333,13 @@ function my_plugin_block_categories( $categories, $post ) {
 			array(
 				'slug' => 'my-category',
 				'title' => __( 'My category', 'my-plugin' ),
+				'icon'  => 'screenoptions',
```

Comment:
I think it's best if we use an example of an SVG here. Over the long term we want to move away from suggesting Dashicons and push folks to use material icons, and this is a good opportunity to do that.

Reply:
I am sure, most of the devs would use custom icons :)

Will update the PR to include SVG or should we both ?

Reply:
I think just an SVG example is okay.

Reply:
Updated